_includes/pr_campaign.html

```diff
@@ -102,3 +102,13 @@
             </div>
     </div>
 </div>
+<div class="container">
```

Comment:
If you want the `container` class for the iframe to blend in you can use `container-fluid` and add `bg-grey` class.

Reply:
@zakibinary ok noted, thanks!